prepDevRelease.js

```diff
@@ -0,0 +1,126 @@
+const fs = require('fs');
+const cp = require("child_process");
+const util = require("util");
+
+let packageJsonPath = './package.json';
+const EXIT_CODE_FATAL_ERROR = 5;
+
+const exec = util.promisify(cp.exec);
+
+function getPackageJson() {
+  if (fs.existsSync(packageJsonPath)) {
+    return JSON.parse(fs.readFileSync(packageJsonPath, { encoding: 'utf8' }));
+  }
+  console.log('FATAL ERROR: package.json path could not be found in the file system.');
+  process.exitCode = EXIT_CODE_FATAL_ERROR;
+  return;
+}
+
+function savePackageJson(packageJson) {
+  fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson));
+}
+
+function getFileVersion(packageJson) {
```

Comment:
NIT: Add a comment describing what this function does like you did for the ones below

Reply:
Do you think it would be good to have this for all of the functions including getPackageJson and savePackageJson? Or do you think those two are clearer on what they're doing there?

Reply:
If it's not expensive, I will also vote in favor of more comments. :)